tests/integration/targets/wafv2_ip_set/tasks/main.yml

```diff
@@ -0,0 +1,227 @@
+---
+- module_defaults:
+    group/aws:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token | default(omit) }}"
+      region: "{{ aws_region }}"
+
+  block:
+    - name: set up aws connection info
+      set_fact:
+        aws_connection_info: &aws_connection_info
+          aws_access_key: "{{ aws_access_key }}"
+          aws_secret_key: "{{ aws_secret_key }}"
+          security_token: "{{ security_token }}"
+          region: "{{ aws_region }}"
+
+    - name: check_mode create ip set
+      wafv2_ip_set:
+        <<: *aws_connection_info
+        name: "{{ ip_set_name }}"
+        state: present
+        description: hallo eins
+        scope: REGIONAL
+        ip_address_version: IPV4
+        addresses:
+          - 8.8.8.8/32
+          - 8.8.4.4/32
+        tags:
+          A: B
+          C: D
+      register: out
+      check_mode: yes
+
+    - name: verify create
+      assert:
+        that:
+          - out is changed
```

Comment:
Could you please assert the return values for the non-check_mode tests?

Reply:
Some are already there: https://github.com/ansible-collections/community.aws/pull/449/files#diff-5c4696663139c385c3dccf20dd0f8354b8685d88dfbed62bc170e0f32ece05abR132  
I've added more checks.